### ISSUE TITLE
Automoderation

### ISSUE THREAD

**Issue by CottageDwellingCat** [CottageDwellingCat/CatCore#2]
- [ ] Regex based matching.
- [ ] Message deletion, flagging, and 'hiding' with webhooks. 
	- [ ] Could share code with the webhook system from #1.
	- [ ] Block proxying messages that would be flagged or hidden.

**Comment by CottageDwellingCat** [comment]
completed, hiding message on hold until after #1 is completed
